stock-plugins/aiverify.stock.accumulated-local-effect/algorithms/README.md

```diff
@@ -0,0 +1,45 @@
+# Algorithm - Accumulated Local Effect
+
+## Description
+* Performs ALE Discrete and ALE Continuous computation
+
+## License
+* Licensed under Apache Software License 2.0
+
+## Developers:
+* AI Verify
+
+## Run Plugin in local
+#### Execute the below bash script in the project root
+```
+#!/bin/bash
+
+# setup virtual environment
+cd aiverify/stock-plugins/
+rm -rf .venv
+python3 -m venv .venv
+source .venv/bin/activate
+
+# execute plugin
```

Comment:
Very small nit, but can we be consistent with the spacing i.e. if you want to include a newline after every code chunk, let's do this:

```
# setup virtual environment
python -m venv .venv
source .venv/bin/activate

# execute plugin
cd aiverify/stock-plugins/aiverify.stock.accumulated-local-effect/algorithms

# install test-engine-core 
pip install -e .'[dev]'
```

Also modified the instructions since the venv does not need to be specific to the stock-plugins folder